integrationTests/src/test/java/org/sonarqube/gradle/GradleTest.java

```diff
@@ -354,7 +354,8 @@ public void testScanAllOnMultiModuleWithSubModulesProjectCollectsTheExpectedSour
         baseDir.resolve("module/submodule/submoduleScript.sh").toString(),
         baseDir.resolve("gradlew").toString(),
         baseDir.resolve("gradlew.bat").toString(),
-        baseDir.resolve("gradle/wrapper/gradle-wrapper.properties").toString()
+        baseDir.resolve("gradle/wrapper/gradle-wrapper.properties").toString(),
+        baseDir.resolve(".gradle/buildOutputCleanup/cache.properties").toString()
```

Comment:
What is the motivation for analyzing this `buildOutputCleanup` directory? Is it a static, committed file or is an artifact introduced by gradle or on the task that ran before the sonar task?

Reply:
As discussed we'll exclude visiting the `.gradle` and `.git` directories